src/utils/v1/terminals.ts

```diff
@@ -10,6 +10,9 @@ const loadTerminalAddress = (
   network: NetworkName,
   terminal: V1TerminalName,
 ): string =>
+  // NOTE: This require is harder to easily change in the code base as it means
```

Comment:
It's also worth nothing that this is v1 so we should try to spend as little time fixing, unless we have to

Reply:
yeah it would be great to fix this eventually, as it results in more efficient code-splitting: https://webpack.js.org/guides/code-splitting/

Should we create a follow-up issue for this?

Reply:
I'm happy to fix it here if you think its valuable - will take a read of code-splitting

Reply:
yep - i'll jiust do it now

Reply:
Actuaslly it is a bigger headache than I originally thought. Since its v1 work, lets create an issue

Reply:
https://github.com/jbx-protocol/juice-interface/issues/1214

Reply:
yep nice